homeassistant/components/rfxtrx.py

```diff
@@ -42,7 +42,7 @@
     ('Total usage', 'W'),
     ('Sound', ''),
     ('Sensor Status', ''),
-    ('Unknown', '')])
```

Comment:
Is it needed?
See also line https://github.com/home-assistant/home-assistant/pull/2974/files/6a91fc3cccb1d022b09f9d27234d0e68fef76357#diff-93645be6c75f881197c9c5a5292e02d6R113
for how an unknown data type is handled now.
I think this is a better way of handle it. To you disagree?